Description: We want to build UTF-8 encoded HTML documents.
Origin: Hilmar Preusse <hille42@web.de>
Forwarded: https://github.com/TeXworks/manual/pull/11
Author: Hilmar Preusse <hille42@web.de>
Last-Update: 20210412

Index: texworks-manual/src/en/html/Makefile
===================================================================
--- texworks-manual.orig/src/en/html/Makefile	2021-04-12 21:33:09.946270122 +0200
+++ texworks-manual/src/en/html/Makefile	2021-04-12 21:33:09.938270140 +0200
@@ -6,7 +6,7 @@
 
 IMAGES = $(Logos) $(EpsFromPdf) $(EpsFromPng)
 
-HTOPTS = "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc" "" "-S*"
+HTOPTS = "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc,charset=utf-8" " -cmozhtf -utf8" "-S*"
 
 # Ghostscript 9.14/9.15 dropped the 'epswrite' output device in favor of the new
 # 'eps2write' output device. Prefer the latter, but fallback to the former for
@@ -23,9 +23,11 @@
 
 html : index.html
 
-index.html : $(IMAGES) $(SOURCES) index.ind manual.cfg tex4ht.env
+index.html : $(IMAGES) $(SOURCES) index.ind manual.cfg
 	@htlatex index $(HTOPTS)
 	@htlatex index $(HTOPTS)
+	#make4ht -u index.tex "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc"
+	#make4ht -u index.tex "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc"
 
 %.tex : ../%.tex
 	$(CP) $< $@
@@ -60,7 +62,7 @@
 	@convert "$<" "$@" $(SUPRESSOUT) 2$(SUPRESSOUT) || convert "$<" ppm:- | pnmtops -noturn | eps2eps - "$@"
 
 
-index.ind : images $(SOURCES) manual.cfg tex4ht.env
+index.ind : images $(SOURCES) manual.cfg
 	@echo "Creating index..."
 	@echo "   running htlatex..."
 	@htlatex index $(HTOPTS)
Index: texworks-manual/src/en/html/tex4ht.env
===================================================================
--- texworks-manual.orig/src/en/html/tex4ht.env	2021-04-12 21:33:09.946270122 +0200
+++ /dev/null	1970-01-01 00:00:00.000000000 +0000
@@ -1,6 +0,0 @@
-Gdvips -E -Ppdf -mode ibmvga -D 10000 -f %%1 -pp %%2  > zzmanual.ps
-Ggs -dSAFER -dBATCH -dNOPAUSE -sDEVICE=pngalpha -r435 -sstdout=%stderr -sOutputFile=- zzmanual.ps | convert -trim +repage -resize 30% -transparent '#FFFFFF' png:- images/%%3
-Grm -f zzmanual.ps
-
-% end of file
-
Index: texworks-manual/src/fr/html/Makefile
===================================================================
--- texworks-manual.orig/src/fr/html/Makefile	2021-04-12 21:33:09.946270122 +0200
+++ texworks-manual/src/fr/html/Makefile	2021-04-12 21:33:09.938270140 +0200
@@ -7,7 +7,7 @@
 
 IMAGES = $(Logos) $(EpsFromPdf) $(EpsFromPng) ${EpsFromJpg}
 
-HTOPTS = "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc" "" "-S*"
+HTOPTS = "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc,charset=utf-8" " -cmozhtf -utf8" "-S*"
 
 # Ghostscript 9.14/9.15 dropped the 'epswrite' output device in favor of the new
 # 'eps2write' output device. Prefer the latter, but fallback to the former for
@@ -24,9 +24,11 @@
 
 html : index.html
 
-index.html : $(IMAGES) $(SOURCES) index.ind manual.cfg tex4ht.env
+index.html : $(IMAGES) $(SOURCES) index.ind manual.cfg
 	@htlatex index $(HTOPTS)
 	@htlatex index $(HTOPTS)
+	#make4ht -u index.tex "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc"
+	#make4ht -u index.tex "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc"
 
 %.tex : ../%.tex
 	$(CP) $< $@
@@ -66,7 +68,7 @@
 	@convert "$<" "$@" $(SUPRESSOUT) 2$(SUPRESSOUT) || convert "$<" ppm:- | pnmtops -noturn | eps2eps - "$@"
 
 
-index.ind : images $(SOURCES) manual.cfg tex4ht.env
+index.ind : images $(SOURCES) manual.cfg
 	@echo "Creating index..."
 	@echo "   running htlatex..."
 	@htlatex index $(HTOPTS)
Index: texworks-manual/src/fr/html/tex4ht.env
===================================================================
--- texworks-manual.orig/src/fr/html/tex4ht.env	2021-04-12 21:33:09.946270122 +0200
+++ /dev/null	1970-01-01 00:00:00.000000000 +0000
@@ -1,6 +0,0 @@
-Gdvips -E -Ppdf -mode ibmvga -D 10000 -f %%1 -pp %%2  > zzmanual.ps
-Ggs -dSAFER -dBATCH -dNOPAUSE -sDEVICE=pngalpha -r435 -sstdout=%stderr -sOutputFile=- zzmanual.ps | convert -trim +repage -resize 30% -transparent '#FFFFFF' png:- images/%%3
-Grm -f zzmanual.ps
-
-% end of file
-
