diff --git a/src/en/html/Makefile b/src/en/html/Makefile
index d990a05..3dec1f8 100644
--- a/src/en/html/Makefile
+++ b/src/en/html/Makefile
@@ -6,7 +6,7 @@ EpsFromPng = images/toolbar1.eps images/toolbar2.eps images/iconTypeset.eps imag
 
 IMAGES = $(Logos) $(EpsFromPdf) $(EpsFromPng)
 
-HTOPTS = "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc" "" "-S*"
+HTOPTS = "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc,charset=utf-8" " -cmozhtf -utf8" "-S*"
 
 # Ghostscript 9.14/9.15 dropped the 'epswrite' output device in favor of the new
 # 'eps2write' output device. Prefer the latter, but fallback to the former for
@@ -23,9 +23,11 @@ SOURCES = index.tex $(filter-out manual.tex,$(subst ../,,$(wildcard ../*.tex ../
 
 html : index.html
 
-index.html : $(IMAGES) $(SOURCES) index.ind manual.cfg tex4ht.env
-	@htlatex index $(HTOPTS)
-	@htlatex index $(HTOPTS)
+index.html : $(IMAGES) $(SOURCES) index.ind manual.cfg
+	@htlatex index $(HTOPTS)
+	@htlatex index $(HTOPTS)
+	#make4ht -u index.tex "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc"
+	#make4ht -u index.tex "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc"
 
 %.tex : ../%.tex
 	$(CP) $< $@
@@ -60,7 +62,7 @@ images/%.eps : ../images/%.png
 	@convert "$<" "$@" $(SUPRESSOUT) 2$(SUPRESSOUT) || convert "$<" ppm:- | pnmtops -noturn | eps2eps - "$@"
 
 
-index.ind : images $(SOURCES) manual.cfg tex4ht.env
+index.ind : images $(SOURCES) manual.cfg
 	@echo "Creating index..."
 	@echo "   running htlatex..."
 	@htlatex index $(HTOPTS)
diff --git a/src/en/html/tex4ht.env b/src/en/html/tex4ht.env
deleted file mode 100644
index eba8725..0000000
--- a/src/en/html/tex4ht.env
+++ /dev/null
@@ -1,6 +0,0 @@
-Gdvips -E -Ppdf -mode ibmvga -D 10000 -f %%1 -pp %%2  > zzmanual.ps
-Ggs -dSAFER -dBATCH -dNOPAUSE -sDEVICE=pngalpha -r435 -sstdout=%stderr -sOutputFile=- zzmanual.ps | convert -trim +repage -resize 30% -transparent '#FFFFFF' png:- images/%%3
-Grm -f zzmanual.ps
-
-% end of file
-
diff --git a/src/fr/html/Makefile b/src/fr/html/Makefile
index 9e28423..55800d8 100644
--- a/src/fr/html/Makefile
+++ b/src/fr/html/Makefile
@@ -7,7 +7,7 @@ EpsFromJpg = images/rechremp.eps
 
 IMAGES = $(Logos) $(EpsFromPdf) $(EpsFromPng) ${EpsFromJpg}
 
-HTOPTS = "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc" "" "-S*"
+HTOPTS = "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc,charset=utf-8" " -cmozhtf -utf8" "-S*"
 
 # Ghostscript 9.14/9.15 dropped the 'epswrite' output device in favor of the new
 # 'eps2write' output device. Prefer the latter, but fallback to the former for
@@ -24,9 +24,11 @@ SOURCES = index.tex $(filter-out manuel.tex,$(subst ../,,$(wildcard ../*.tex ../
 
 html : index.html
 
-index.html : $(IMAGES) $(SOURCES) index.ind manual.cfg tex4ht.env
-	@htlatex index $(HTOPTS)
-	@htlatex index $(HTOPTS)
+index.html : $(IMAGES) $(SOURCES) index.ind manual.cfg
+	@htlatex index $(HTOPTS)
+	@htlatex index $(HTOPTS)
+	#make4ht -u index.tex "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc"
+	#make4ht -u index.tex "manual,info,2,imgdir:images/,sec-filename,next,index=2,url-enc"
 
 %.tex : ../%.tex
 	$(CP) $< $@
@@ -66,7 +68,7 @@ images/%.eps : ../images/%.jpg
 	@convert "$<" "$@" $(SUPRESSOUT) 2$(SUPRESSOUT) || convert "$<" ppm:- | pnmtops -noturn | eps2eps - "$@"
 
 
-index.ind : images $(SOURCES) manual.cfg tex4ht.env
+index.ind : images $(SOURCES) manual.cfg
 	@echo "Creating index..."
 	@echo "   running htlatex..."
 	@htlatex index $(HTOPTS)
diff --git a/src/fr/html/tex4ht.env b/src/fr/html/tex4ht.env
deleted file mode 100644
index eba8725..0000000
--- a/src/fr/html/tex4ht.env
+++ /dev/null
@@ -1,6 +0,0 @@
-Gdvips -E -Ppdf -mode ibmvga -D 10000 -f %%1 -pp %%2  > zzmanual.ps
-Ggs -dSAFER -dBATCH -dNOPAUSE -sDEVICE=pngalpha -r435 -sstdout=%stderr -sOutputFile=- zzmanual.ps | convert -trim +repage -resize 30% -transparent '#FFFFFF' png:- images/%%3
-Grm -f zzmanual.ps
-
-% end of file
-
